# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/TransceiverImpl.cpp
# Commit: afe92bb8ee8e
# Full Hash: afe92bb8ee8e54127cbe8ae06356974d7841ff90
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Small fixes to RTCRtpReceiver and TransceiverImpl. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130068
# ==============================================================================

diff -r 1a244e507326 -r afe92bb8ee8e dom/media/webrtc/jsapi/TransceiverImpl.cpp
--- a/dom/media/webrtc/jsapi/TransceiverImpl.cpp	Wed Nov 04 07:25:46 2020 -0500
+++ b/dom/media/webrtc/jsapi/TransceiverImpl.cpp	Wed Nov 04 11:10:31 2020 -0500
@@ -844,17 +844,17 @@
   }
 
   dom::MediaSourceEnum source = videotrack->GetSource().GetMediaSource();
-  webrtc::VideoCodecMode mode = webrtc::kRealtimeVideo;
+  webrtc::VideoCodecMode mode = webrtc::VideoCodecMode::kRealtimeVideo;
   switch (source) {
     case dom::MediaSourceEnum::Browser:
     case dom::MediaSourceEnum::Screen:
     case dom::MediaSourceEnum::Window:
-      mode = webrtc::kScreensharing;
+      mode = webrtc::VideoCodecMode::kScreensharing;
       break;
 
     case dom::MediaSourceEnum::Camera:
     default:
-      mode = webrtc::kRealtimeVideo;
+      mode = webrtc::VideoCodecMode::kRealtimeVideo;
       break;
   }
 
